src/data/fintype/basic.lean

```diff
@@ -845,9 +845,13 @@ lemma univ_option (α : Type*) [fintype α] : (univ : finset (option α)) = inse
 def fintype_of_option {α : Type*} [fintype (option α)] : fintype α :=
 ⟨finset.erase_none (fintype.elems (option α)), λ x, mem_erase_none.mpr (fintype.complete (some x))⟩
 
+/-- A type is a `fintype` if it's the successor (using `option`) of a `fintype`. -/
+def fintype_of_equiv_option [fintype α] (f : option α ≃ β) : fintype β :=
+fintype.of_equiv _ f
```

Comment:
What's the point of this definition given it's just a special case of `of_equiv`, which is also a shower spelling?